### PR TITLE
DOCUMENTATION: Fix powerdns config keys using camel case

### PR DESCRIPTION
**Pull request by afrimberger**

### CODE REVIEW

**Comment by tlimoncelli** [comment]
CC @jpbede

**Review by tlimoncelli — APPROVED** [review]

File: docs/_providers/powerdns.md [hunk 1]
```diff
@@ -54,4 +54,4 @@ See the [PowerDNS documentation](https://doc.powerdns.com/authoritative/http-api
 
 ## Caveats
```
Comment (jpbede) [inline docs/_providers/powerdns.md:55]: This can be removed. On-demand activation/deactivation has been implemented with AUTODNSSEC.

**Review by jpbede — CHANGES_REQUESTED** [review]

**Comment by jpbede** [comment]
@afrimberger thanks for spotting this. I've added a comment.

**Comment by afrimberger** [comment]
See #960

**Pull request closed without merging**